### PR TITLE
Added MIT License

### LINKED ISSUE #26: Add License to the Project
Adding MIT License to the project

### PR DESCRIPTION
**Pull request by AdityaTeltia**
Signed-off-by: AdityaTeltia <teltia.aditya22@gmail.com>

### Issue Number

fixes #26 

### Describe the changes you've made
Added MIT License 

### Describe if there is any unusual behavior (Any Warning) of your code(Write `NA` if there isn't)
NA

### Test plan (OPTIONAL)

A good test plan should give instructions that someone else can easily follow.

### Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
   "name": "educult",
   "version": "0.1.0",
   "private": true,
+  "licence": "MIT",
```
Comment (Abhishek-kumar09) [inline package.json:5]: Is it the correct spelling?

Comment (AdityaTeltia) [reply package.json:5]: Yes ! I rechecked it

Comment (Abhishek-kumar09) [reply package.json:5]: Where?

Comment (Abhishek-kumar09) [reply package.json:5]: Okay, I see. Nice

**Comment by kaiwalyakoparkar** [comment]
I think that should be LICENCE.md rather than LICENCE.txt

**Comment by AdityaTeltia** [comment]
> I think that should be LICENCE.md rather than LICENCE.txt

Both Works you can refer[ here](https://choosealicense.com/licenses/mit/)

**Pull request merged**